jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
```

Comment:
Because this is more flexible. `upgrade-versions` is (currently) only run when we want to _upgrade_ the versions of a few selected packages.

By having a separate update-versions, we are not tied to the use cases of upgrade-versions. This allows us to update the README whenever a package that is not targeted by the `upgrade-versions` script is changed.